f5_sphinx_theme/static/js/clouddocs.js

```diff
@@ -9,6 +9,61 @@ $(document).ready(function () {
 
 
 
+//Version Warning Banner
+
+function renderVersionWarning(){
+
+    $.getJSON( "../../versions.json", function(vesrionsJson) {
+
+        var pathes = document.location.pathname.split("/");
+        pathes.pop();
+
+        var versionUrlPathes = vesrionsJson.latestVersion.url.split("/");
+        versionUrlPathes.pop();
+
+        var currentPath = pathes.slice(0, versionUrlPathes.length).join("/") + "/";
+
+        if(currentPath != vesrionsJson.latestVersion.url){
+
+            for (var index in vesrionsJson.otherVersions){
+
+                if(vesrionsJson.otherVersions[index].url == currentPath){
+
+                    $("#version-warning").show();
+                    $("#currentVersion").text("This content applies to " + vesrionsJson.otherVersions[index].name);
```

Comment:
Is versions supposed to be spelled vesrions on this line?

Reply:
ohh - you are right! I mis-spelled "versionsJson"; I am going to fix it now

Reply:
this is fixed now! just committed the change